Tests/Admin/AdminTest.php

```diff
@@ -1524,18 +1524,6 @@ public function testGetSubject()
         $admin->setModelManager($modelManager);
 
         $admin->setRequest(new Request(array('id' => 'azerty')));
-        $this->assertFalse($admin->getSubject());
```

Comment:
See the line above, the behavior is changed because azerty is not considered invalid anymore.